### PR TITLE
Localization: Update logs level in map creation.

### PR DESCRIPTION
**Pull request by zhouyao4321**
Corresponding issue:
http://newicafe.baidu.com/issue/IDG-Apollo-3657/show?from=page

### CODE REVIEW

File: modules/localization/msf/common/io/velodyne_utility.cc [hunk 1]
```diff
@@ -151,7 +151,7 @@ void LoadPosesAndStds(const std::string& file_path,
       timestamps->push_back(timestamp);
 
       Eigen::Vector3d std;
-      std << std_x, std_y, std_z;
+      AINFO << std_x, std_y, std_z;
       stds->push_back(std);
```
Comment (jinghaomiao) [inline modules/localization/msf/common/io/velodyne_utility.cc]: Please make sure this logic is correct. The original implementation sets values for a vector. 

Also please make sure that AINFO is only used for initialization/check purpose. Other runtime info should be used together with ADEBUG.

File: modules/localization/msf/local_tool/map_creation/lossless_map_creator.cc [hunk 2]
```diff
@@ -75,16 +76,16 @@ bool ParseCommandLine(int argc, char* argv[],
...
+      AERROR << desc;
       return false;
     }
     boost::program_options::notify(*vm);
   } catch (std::exception& e) {
-    std::cerr << "Error" << e.what() << std::endl;
-    std::cerr << desc << std::endl;
+    AERROR << "Error" << e.what();
+    AERROR << desc;
     return false;
```
Comment (jinghaomiao) [inline modules/localization/msf/local_tool/map_creation/lossless_map_creator.cc]: Combine the error message.

File: modules/localization/msf/local_tool/map_creation/lossless_map_creator.cc [hunk 3]
```diff
@@ -246,8 +246,8 @@ int main(int argc, char** argv) {
     }
     fclose(file);
   } else {
-    std::cerr << "Can't open file: "
-              << "./lossless_map/config.txt" << std::endl;
+    AERROR << "Can't open file: "
+           << "./lossless_map/config.txt";
```
Comment (jinghaomiao) [inline modules/localization/msf/local_tool/map_creation/lossless_map_creator.cc]: Combine two lines.

File: modules/localization/msf/local_tool/map_creation/lossless_map_to_lossy_map.cc [hunk 4]
```diff
@@ -48,7 +49,7 @@ MapNodeIndex GetMapIndexFromMapFolder(const std::string& map_folder) {
   if (zone == "south") {
     index.zone_id_ = -index.zone_id_;
   }
-  std::cout << index << std::endl;
+  AINFO << index;
```
Comment (jinghaomiao) [inline modules/localization/msf/local_tool/map_creation/lossless_map_to_lossy_map.cc]: Use ADEBUG if appropriate. It applies to all.

File: modules/localization/msf/common/io/velodyne_utility.cc (hunk 1 above)
Comment (zhouyao4321) [reply modules/localization/msf/common/io/velodyne_utility.cc]: Fixed.

File: modules/localization/msf/local_tool/map_creation/lossless_map_creator.cc (hunk 2 above)
Comment (zhouyao4321) [reply modules/localization/msf/local_tool/map_creation/lossless_map_creator.cc]: Fixed.

File: modules/localization/msf/local_tool/map_creation/lossless_map_creator.cc (hunk 3 above)
Comment (zhouyao4321) [reply modules/localization/msf/local_tool/map_creation/lossless_map_creator.cc]: Fixed.

File: modules/localization/msf/local_tool/map_creation/lossless_map_to_lossy_map.cc (hunk 4 above)
Comment (zhouyao4321) [reply modules/localization/msf/local_tool/map_creation/lossless_map_to_lossy_map.cc]: Fixed.

**Pull request merged**